src/pydata_sphinx_theme/assets/styles/sections/_sidebar-primary.scss

```diff
@@ -2,17 +2,21 @@
  * The primary sidebar on the left.
  * e.g., between-pages navigation.
  */
-.bd-sidebar {
+.bd-sidebar-primary {
```

Comment:
I'm +1 on this, but just a note that this will be a breaking change so we should note it in the release notes